src/spas/libs/member-management/feature/src/lib/member-management/user-detail/user-detail.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { User } from '../../../../../domain/src/lib/models/user-profile';
+
+@Component({
+  selector: 'jwadmin-user-detail',
```

Comment:
jwadmin prefix can be removed, feature components should be prefixed with the name of the feature. In this case, this selector should be ```selector: 'member-management-user-detail',```